etc/esp-idf/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+## IDF Component Manager Manifest File
+dependencies:
+  OpenMRN:
```

Comment:
Close, we can comment out this OpenMRN section as it would create a circular dependency which may trip up the dependency logic in esp-idf.